run_combination_test.py

```diff
@@ -71,6 +71,7 @@ def get_shuffle_params(params, index):
     if params['h5py'] == '2.6':
         conf['requires'].append('h5py<2.7')
     elif params['h5py'] == '2.5':
+        conf['requires'].append('numpy<1.10')
```

Comment:
Sorry, i didn't understand it yet. Ver 2.5.0 also installs numpy doesn't it? https://github.com/h5py/h5py/blob/2.5.0/setup.py#L137